packages/client/src/styles/globals.css

```
@@ -2701,6 +2701,392 @@ body {
+  76%,
+  100% {
+    opacity: 0;
+  }
+  73%,
+  75% {
+    opacity: 1;
+  }
+}
+
+@keyframes mari-home-professor-map {
+  from {
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)